test/e2e/app-dir/next-form/default/next-form-prefetch.test.ts

```diff
@@ -0,0 +1,257 @@
+import { nextTestSetup, isNextDev } from 'e2e-utils'
+import {
+  NEXT_ROUTER_PREFETCH_HEADER,
+  NEXT_RSC_UNION_QUERY,
+  RSC_HEADER,
+} from 'next/src/client/components/app-router-headers'
+import type { Route, Page, Request as PlaywrightRequest } from 'playwright'
+import { WebdriverOptions } from '../../../../lib/next-webdriver'
+import { retry } from '../../../../lib/next-test-utils'
+
+const _describe =
+  // prefetching is disabled in dev.
+  isNextDev ||
+  // FIXME(form): currently failing in PPR, unflag this when https://github.com/vercel/next.js/pull/70532 is ready
+  process.env.__NEXT_EXPERIMENTAL_PPR
+    ? describe.skip
+    : describe
```

Comment:
This is waiting on https://github.com/vercel/next.js/pull/70532. The bug was there before, we just didn't have a test that ran into it, so it doesn't make sense to block this feature